docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.8'
+services:
+  node:
+    image: node:20-alpine
+    restart: always
+    working_dir: /
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./backend:/server
+
+    command: [sh, -c, "cd server && npm install && npx ts-node-dev --respawn ./server/index.ts"]
```

Comment:
you may replace it by "npm run start:docker" command. And if you need to change something in the future, it will be in more predictable place